SU-HSDO/suhumsci
# STN-100: Exposed Filters

# READY FOR REVIEW

## Summary
This pull request adds styles for exposed filters. It creates foundational work for:
- input styles
- chosen.js dropdown styles
- button styles
- exposed filter layout

Assets:
- [Design](https://www.figma.com/file/toNZnFLadtpNpG5dPzNGWz/%E2%9A%99%EF%B8%8F-Colorful-Theme---Working?node-id=2067%3A0)
- [Jira STN-100](https://sparkbox.atlassian.net/browse/STN-100)

Note: This first pass at exposed filters does not include the reset button or the custom dropdown default text that is in the designs. [STN-153](https://sparkbox.atlassian.net/browse/STN-153) which will probably be prioritized when we build the courses will handle those changes.

✅ Received design approval from @meranicosme 

Styles have been tested with:
- Chrome
- Firefox
- Safari
- Edge
- IE 11
- Voice Over

## Need Review By (Date)
02/03

## Urgency
medium

## Steps to Test
1. Run `npm test` and confirm that all tests and linters pass.
1. Import the horizontal card view: [horizontal-card-config.txt](https://github.com/SU-HSDO/suhumsci/files/4149105/horizontal-card-config.txt)
1. Create a basic page with a view using the horizontal card with list display:
    <img width="276" alt="Screen Shot 2020-02-03 at 12 17 19 PM" src="https://user-images.githubusercontent.com/2486846/73674767-3445c500-467f-11ea-95ab-2d9eabb564db.png">
1. Confirm that exposed filters:
    - [ ] Are responsive
    - [ ] Are keyboard accessible
    - [ ] Match the design (except for the differences noted above)
   

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
